config/cinder/templates/cinder-db-sync.sh.j2

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+set -e
+cinder-manage db sync
+
```

Comment:
why not jut place "cinder-manage db sync" in COMMAND env and get rid this file ?